GrandCentralBoard/Widgets/Harvest/View/AreaBarDashedLineWithLabel.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Created by Oktawian Chojnacki on 14.04.2016.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import UIKit
+
+
+final class AreaBarDashedLineWithLabel : UIView {
+
+    @IBOutlet private var valueLabelRight: UILabel!
+    @IBOutlet private var valueLabelLeft: UILabel!
+    private var color: UIColor?
+
+    func configureWithViewModel(viewModel: AreaBarItemViewModel) {
+
+        color = viewModel.color
+        setNeedsDisplay()
+
+        switch viewModel.valueLabelMode {
+        case .Hidden:
+            valueLabelLeft.hidden = true
+            valueLabelRight.hidden = true
+        case .Left(let text):
+            valueLabelLeft.textColor = color
+            valueLabelLeft.hidden = false
+            valueLabelLeft.text = text
+            valueLabelRight.hidden = true
+        case .Right(let text):
+            valueLabelRight.textColor = color
+            valueLabelRight.hidden = false
+            valueLabelRight.text = text
+            valueLabelLeft.hidden = true
+        }
+    }
+
+    // MARK: - fromNib
+
+    class func fromNib() -> AreaBarDashedLineWithLabel {
```

Comment:
I've seen this method in a lot of different places. Maybe we could refactor it out to a UIView extension